tests/test_nb.py

```diff
@@ -0,0 +1,103 @@
+import os
+import pathlib
+import subprocess
+import shutil
+from unittest import mock
+
+import pytest
+
+import jupyter_client.kernelspec
+
+import json
+import sys
+
+PROJECT_ROOT = pathlib.Path(__file__).parent.absolute().parent
+NB_FILE_STD = pathlib.Path(__file__).parent.absolute() / 'notebook-std.ipynb'
+NB_FILE_VIP = pathlib.Path(__file__).parent.absolute() / 'notebook-vip.ipynb'
+
+
+def test_nb_vip_no_venv(tmp_path):
+    from vip_ipykernel.kernelspec import install as install_vip
+    import vip_ipykernel.kernelspec
+
+    os.chdir(tmp_path)
```

Comment:
Use the [monkeypatch fixture](https://docs.pytest.org/en/latest/monkeypatch.html) to avoid chaning the cwd for later tests?

Reply:
Aha, clever! Also didn't know that was a thing, thanks :D